scout/constants/case_tags.py

```diff
@@ -1,10 +1,11 @@
-ANALYSIS_TYPES = ('wgs', 'wes', 'mixed', 'unknown')
+ANALYSIS_TYPES = ('wgs', 'wes', 'mixed', 'unknown', 'panel')
 
 SEX_MAP = {1: 'male', 2: 'female', 'other': 'unknown', 0: 'unknown', 
           '1': 'male', '2': 'female', '0': 'unknown'}
 REV_SEX_MAP = {'male':'1', 'female': '2', 'unknown': '0'}
 
 PHENOTYPE_MAP = {1: 'unaffected', 2: 'affected', 0: 'unknown', -9: 'unknown'}
+CANCER_PHENOTYPE_MAP = {1: 'normal', 2: 'tumor', 0: 'unknown', -9: 'unknown'}
```

Comment:
This field will only communicate if the sample is from a tumor or not. For mor specific information we can use tumor_type and the other stuff that you suggested above.
Since this is information from a PED format file we only have three options: affected, unaffected and unknown for this field

Reply:
Ok!